website/docs/plugin/best-practices/hashicorp-provider-design-principles.mdx

```diff
@@ -79,3 +79,14 @@ The benefits of this practice include:
 - Ensuring Terraform can statically validate the entire configuration anywhere
 - Preventing practitioner issues should the environment change between Terraform commands
 - Preventing practitioner issues should networking or a service become unavailable
+
+## Ephemeral Resources should represent a single sensitive API object
+
+A Terraform ephemeral resource should be a declarative representation of a single sensitive API object, such as an API token or secret, which will be
```

Comment:
I think it'd make more sense to not limit our description of this to sensitive data (despite that being a large use-case), maybe something like:

```suggestion
A Terraform ephemeral resource should be a declarative representation of a single API object that doesn't require Terraform to store it's data between runs, such as sensitive data like an API token or secret. This ephemeral data is created and/or retrieved from the API during the Open operation.
```